docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -12,5 +12,259 @@ but the format and content is still evolving.
 We hope you will contribute your experiences
 and questions that you have.
 
-No two migrations are alike but these are some general guidelines
-for how to approach the project:
+The Keptn Lifecycle Toolkit uses a different paradigm
+than Keptn v1 and so migration is not a straight-forward process.
+Much of the logic and functionality of your Keptn v1 projects
+can be migrated to KLT
+but must be rewritten to utilize KLT components.
+
+## How many namespaces?
```

Comment:
I quite like this section! I am wondering if it should be the first one, probably as a v1 user I immediately care about project and services , than SLOs, and only at the very end, when I have an idea of what is happening, about how to allocate stuff in namespaces

Reply:
I put the namespaces stuff first because I was thinking this is something they need to address when they set up their cluster and before the install KLT.  But I'm thinking that most of this should really be moved to the general KLT installation section where we don't currently discuss namespaces at all.